apps/devhub/views.py

```diff
@@ -1777,8 +1777,23 @@ def api_key(request):
         new_credentials = APIKey.new_jwt_credentials(request.user)
         log.info('new JWT key created: {}'.format(new_credentials))
 
+        send_key_change_email(request.user.email, new_credentials.key)
```

Comment:
hmm, @magopian, does this need `async=True` so that it gets sent in a background celery task? I suppose we could leave it as and see if it takes too long. When sending synchronously, the page response will be blocked until the email is sent.